### DISCUSSION TITLE
Camera2 height adjustment

### DISCUSSION THREAD

**Discussion by LukyTemy** [kinsi55/CS_BeatSaber_Camera2#58] [Q&A] [answered]
May I ask you guys, how to change height of the Camera2 in FirstPerson mode? When I want to record the gameplay, the camera appears a little bit higher than normal because the notes are lower than usual, can I somehow change the height of the camera?

**Comment by kinsi55** [comment, marked as answer]
This is not exposed in the settings UI because its a pretty niche thing, but as pointed out in [the FAQ in the Wiki](https://github.com/kinsi55/CS_BeatSaber_Camera2/wiki#faq), you can manually adjust the Camera offset / position in the respective cameras configs [targetPos / targetRot](https://github.com/kinsi55/CS_BeatSaber_Camera2/wiki/Camera-Configuration#targetpos--targetrot) - Height would be y in targetPos.

Keep in mind that if you make use of this, you probably want to disable the "Pivoting offset" option, depending on how much you adjust it by.

**Reply by LukyTemy** [reply]
Does it work for FirstPerson camera too? I thought that targetPos only works for Positional camera and not the FirstPerson one

**Reply by kinsi55** [reply]
Settings that dont affect the specific camera type are omitted from the config, so yes they work (For FP its an offset, for TP its a position). Those are the best names I could come up with at the time and they are now set in stone :D
